script/Arcueid/Day 7/07_00_ARC07_3E.txt

```diff
@@ -374,7 +374,7 @@ I can't even muster a response to Arcueid's words.
 [sha:3999e5e052dac5fb8a7966753c9085c761ed240a]{
 -- Page 299, Offset 17308.
 -- 今は吐き気を堪えるだけで精一杯だ。
-Right now, it's taking all I've got just to stop myself from throwing up.
+Right now, it's taking all that I've got just to stop myself from throwing up.
```

Comment:
I've been trying to drop `that`s where possible so that @IrishKnight0 doesn't come after me :aokoheh: